src/initialize-custom-elements.ts

```diff
@@ -1,54 +1,12 @@
 import Application from '@glimmer/application';
+import glimmerElementFactory from './glimmer-element';
 
-export default function initializeCustomElements(app: Application, customElementDefinitions: string[]): void {
-  customElementDefinitions.forEach((name) => {
-    initializeCustomElement(app, name);
+export function initializeCustomElements(app: Application, customElementDefinitions: string[]): void {
+  customElementDefinitions.forEach(name => {
+    initializeCustomElement(app, name, name, []);
```

Comment:
Should `initializeCustomElements` accept customElementDefinitions as an array of objects that have both a `name` and `trackedAttributes` key? Right now, it looks like we watch for no attributes at all.